docs/glossary.md

```diff
@@ -209,12 +209,6 @@ Learn more about the General Availability release stage
 
 _Tags: [product-release-stages](/tags/product-release-stages), [term](/tags/term)_
 
-#### [Global Namespace](/namespaces#global-namespace)
```

Comment:
Instead of deleting it - lets put "This term is deprecated and is now called "Multi-region Namespace" and point to the new term.